botorch_community/models/vbll_helper.py

```diff
@@ -412,9 +412,26 @@ def __init__(
 
         # last layer distribution
         self.W_dist = get_parameterization(parameterization)
-        self.W_mean = nn.Parameter(
-            torch.randn(out_features, in_features, dtype=self.dtype)
-        )
+
+        if mean_initialization is None:
+            self.W_mean = nn.Parameter(
+                torch.zeros(out_features, in_features, dtype=self.dtype)
+            )
```

Comment:
As this changes the default from `torch.randn` to `torch.zeros` this does change the default behavior - is this intended?